lib/ExpensesPage/expensesPage.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:percent_indicator/linear_percent_indicator.dart';
+import 'package:budgie/BudgetCategories/budgetCategoryPrice.dart';
+import 'package:budgie/ExpensesPage/expensesContainer.dart';
```

Comment:
this also uses the full path, I don't think it's a big deal but maybe worth mentioning